anime_downloader/sites/helpers/selescrape.py

```diff
@@ -2,88 +2,88 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from anime_downloader.const import get_random_header
 from selenium.webdriver.common.by import By
+import requests, os, click, time, json
 from urllib.parse import urlencode
 from urllib.parse import urlsplit
 from selenium import webdriver
 from bs4 import BeautifulSoup
 from logging import exception
 from sys import platform
-import requests, os
-import click
-import time
-import json
 
-header = get_random_header()
 def test_os():
-    if platform.startswith("linux"):
+    if platform.startswith("lin"):
         os_name = "linux"
-    elif platform == "darwin":
+    elif platform.startswith("da"):
```

Comment:
why? why not `==`

Reply:
Well, no real reason. Just thought, what if the platform name getting returned isn't the one I haven't redefined? Then boom and error. But that is just an extreme scenario I thought of.